Trippy/API/FetchSavedHandler.h

```diff
@@ -0,0 +1,30 @@
+//
+//  FetchSavedHandler.h
+//  Trippy
+//
+//  Created by Catherine Lu on 7/7/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Parse/Parse.h"
+#import "Collection.h"
+#import "Location.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol FetchSavedHandlerDelegate
+- (void) addFetchedCollection:(Collection *)collection;
+- (void) addFetchedLocation:(Location *)location;
+- (void) postedCollectionSuccess:(Collection *)collection;
+- (void) generalRequestFail:(NSError *)error;
+@end
+
+@interface FetchSavedHandler : NSObject
```

Comment:
I think there's another opportunity for a better name here too. For one, it sounds like you're fetching saved handlers, when actually it's the handler that fetches saved [data]. However, it also writes data (in the post methods). So, I would suggest calling it the `CacheDataHandler` ('cache' is cooler than 'save' 😎).